package.json

```diff
@@ -11,7 +11,8 @@
     "d3-ease": "^1.0.3",
     "exenv": "^1.2.0",
     "prop-types": "^15.6.0",
-    "react-move": "^6.1.0"
+    "react-move": "^6.1.0",
+    "wicg-inert": "^3.0.3"
```

Comment:
`wicg-inert` v3 has es6 code  in src/inert.js. which used by module in package.json
see https://github.com/WICG/inert/blob/master/package.json#L6

Reply:
Just to add, this breaks IE11 even with Polyfills.